src/components/Charts/Groupings.tsx

```diff
@@ -0,0 +1,260 @@
+        renderTabLabel: (metricValue, projections) => (
+          <ChartTab metricName="ICU patients" metricValueInfo={metricValue} />
+        ),
+        renderChart: projections => (
+          <SingleLocationChartContainer
+            metric={ExploreMetric.ICU_HOSPITALIZATIONS}
+            projections={projections}
+          />
+        ),
+      },
+      {
+        metric: ExploreMetric.HOSPITALIZATIONS,
+        metricType: MetricType.EXPLORE_METRIC,
+        renderTabLabel: (metricValue, projections) => (
+          <ChartTab
+            metricName="Hospitalized patients"
+            metricValueInfo={metricValue}
+          />
+        ),
+        renderChart: projections => (
+          <SingleLocationChartContainer
+            metric={ExploreMetric.HOSPITALIZATIONS}
+            projections={projections}
+          />
+        ),
+      },
+    ],
+  },
+  {
+    groupHeader: GroupHeader.DEATHS,
+    metricList: [
+      {
+        metric: ExploreMetric.DEATHS,
+        metricType: MetricType.EXPLORE_METRIC,
+        renderTabLabel: (metricValue, projections) => (
+          <ChartTab metricName="Daily deaths" metricValueInfo={metricValue} />
+        ),
+        renderChart: projections => (
+          <SingleLocationChartContainer
+            metric={ExploreMetric.DEATHS}
+            projections={projections}
+          />
+        ),
+      },
+    ],
+  },
+];
+
+export const nullValueString = '--';
+
+export function getValueInfo(
+  stats: MetricValues,
+  metricItem: MetricChartInfo,
+  projections: Projections,
+): ValueInfo {
+  const { metric, metricType } = metricItem;
+  if (metricType === MetricType.KEY_METRIC) {
+    const statValue = stats[metric as Metric];
+    const levelInfo = getLevelInfo(metric as Metric, statValue);
+    const formattedValue = formatValue(
+      metric as Metric,
+      statValue,
+      nullValueString,
+    );
+    return {
+      unformattedValue: statValue,
+      formattedValue,
+      levelColor: levelInfo.color,
+    };
+  } else {
+    const smoothedSeries = getAveragedSeriesForMetric(
+      metric as ExploreMetric,
+      projections.primary,
+      '',
+      false,
+    );
+    const hasData = some([smoothedSeries], ({ data }) => data.length > 0);
+    if (!hasData) {
+      return { unformattedValue: null, formattedValue: nullValueString };
+    } else {
+      const data = smoothedSeries.data.filter(data => Number.isFinite(data.y));
+      const lastPoint = last(data);
+      return {
+        unformattedValue: lastPoint.y,
+        formattedValue: formatDecimal(lastPoint.y, 1),
+      };
+    }
+  }
+}
+
+// Used for scrolling to correct chart group when clicking summary stat
+export const getChartGroupFromMetric = (
+  metricToScrollTo: Metric,
+): ChartGroup | null => {
+  const groupWithMetric = find(
+    CHART_GROUPS,
+    group =>
+      find(group.metricList, metric => metric.metric === metricToScrollTo) !==
+      undefined,
+  );
```

Comment:
Maybe it would be easier if `CHART_GROUPS` was a map instead, but this works too!